app/services/subtitle_processing.py

```
@@ -0,0 +1,125 @@
+            translated_text_from_aws_translate=self.get_translation_via_aws_translate(text_from_vision_api,"ja","en")
+            language_mapping_list.append({"start_time":start_time,"end_time":end_time,"original_text":text_from_vision_api,"translated_text":translated_text_from_aws_translate})
+            start_time=start_time+1
+            end_time=end_time+1
+        logging.info(f"Step 3 completed successfully")
+        logging.info(f"We are Sanitising the received JSON now")
+        logging.info(f"Starting Step 4: Sanitising the JSON")
+        for i in range(0,len(language_mapping_list)):
+            if language_mapping_list[i]["translated_text"]=="":
+                language_mapping_list[i]["translated_text"]=language_mapping_list[i-1]["translated_text"]
+                language_mapping_list[i]["original_text"]=language_mapping_list[i-1]["original_text"]
+        logging.info(f"Step 4 completed successfully")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)